docker-image-tests/ps/settings.py

```diff
@@ -95,6 +95,7 @@
     ps_binaries = ps57_binaries
     ps_plugins = ps57_plugins
     ps_functions = ps57_functions
+    ps_components = ps80_components
```

Comment:
Why do we add "ps80_components" to ps 5.7 part?

Reply:
IF we don't define than it throws error :

```
==================================== ERRORS ====================================
__________________ ERROR collecting tests/test_ps_dynamic.py ___________________
tests/test_ps_dynamic.py:24: in <module>
    class TestDynamic:
tests/test_ps_dynamic.py:56: in TestDynamic
    @pytest.mark.parametrize("cmpt", ps_components)
E   NameError: name 'ps_components' is not defined
```